README.md

```diff
@@ -1,14 +1,84 @@
+
+In further research, we built a data tool that matches visitor homes to show homes (still using model data) while considering maximum driving distance and capacity of the show homes. The mock example below shows how certain visitor homes (gray) cannot be matched with a suitable host home (blue), either because the host home is too far away or because it's already booked out - in this simple example, each show home can only host three visitors.
+
+<img src="./docs/mock_network_example.png"  width="400"  height="300">
+
+The data tool lets you explore different scenarios and compute the impact and reach of a network. For example, you can set the number of visitors per slot, adjust the maximum driving distance or play around with different numbers of hosts and visitors.
+
+We found different patterns for different areas and property types when modelling a potential network, suggesting that show home networks may not work equally well in all areas and may require the addition of strategically placed show homes in remote areas or show rooms (e.g. in DIY stores) in regions with high demand.
+
+<!-- <p float="left">
+<img src="./docs/network_example_1.png"  width="250"  height="180">
+<img src="./docs/network_example_2.png"  width="250"  height="180">
+</p>
+
+<p float="left">
+<img src="./docs/network_example_3.png"  width="250" height='180'>
+<img src="./docs/network_example_4.png"  width="220"   height="180">
+</p> -->
+
+<img src="./docs/network_examples.png"  width="800"  height="580">
+
+Future research may include an agent-based model that investigates and models how a network would develop over time in different areas.
+
+---
+
+## Setup and Troubleshooting
 
 - Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
   - Install: `git-crypt`, `direnv`, and `conda`
   - Have a Nesta AWS account configured with `awscli`
-- Run `make install` to configure the development environment:
-  - Setup the conda environment
-  - Configure pre-commit
-  - Configure metaflow to use AWS
+
+```
+git clone https://github.com/nestauk/show_homes.git
+cd show_homes
+git checkout 4_show_home_network
+make install
+```
+
+`make install` may take a while and and could potentially throw an error for AWS+Metaflow like below, but that's fine.
+
+```
+May throw error at megaflop Configuring Metaflow + AWSAWS + Metaflow setup failed, check .cookiecutter/state/setup-metaflow for more info
+make: *** [.cookiecutter/state/setup-metaflow] Error 1
+```
+
+Next, activate the environment and download the necessary files:
+
+```
+conda activate show_homes
+aws s3 sync s3://asf-show-homes/inputs ./inputs
+aws s3 sync s3://asf-show-homes/outputs/data ./outputs/data
```

Comment:
Unfortunately, we need the input data (epc_for_show_homes.csv) for the visualisations as well. But I have removed the intermediate step data dump(epc_df_with_mcs.csv) so it's less than 4GB in total now. I'll see if this can be further improved in the future.